File: Patterns/Java/Creational/Builder/Outfit.java
import java.util.ArrayList;
import java.util.List;

public class Outfit{

    private List<Clothe> shoped;

    Outfit(){
        shoped = new ArrayList<Clothe>();
    }

    public void addClothe(Clothe clothe){
        shoped.add(clothe);
    }

    public void listClothe(){
        System.out.println("Outfit: ");
        for(Clothe clothe: shoped)
            System.out.println( "\t" + clothe.getName() );
    }

}

File: Patterns/C#/Creational/Builder/Outfit.cs
using System;
using System.Collections.Generic;

public class Outfit{

    private List<Clothe> shoped;

    public Outfit(){
        shoped = new List<Clothe>();
    }

    public void addClothe(Clothe clothe){
        shoped.Add(clothe);
    }

    public void listClothe(){
        Console.WriteLine("Outfit: ");
        foreach(Clothe clothe in shoped)
            Console.WriteLine("\t" + clothe.getName() );
    }

}
